homeassistant/components/webostv/__init__.py

```diff
@@ -120,7 +129,9 @@ def read_client_keys(config_file):
     table = db.Table("unnamed", db.MetaData(), autoload=True, autoload_with=engine)
     results = engine.connect().execute(db.select([table])).fetchall()
     client_keys = {k: loads(v) for k, v in results}
-    return client_keys
+    if isinstance(client_keys, dict):
```

Comment:
We create the dict on the line above so this check shouldn't be needed.

Reply:
Now realized it is just mypy complains because the same variable name is used for reading from json, Thanks!